source/NVDAObjects/IAccessible/ia2Web.py

```diff
@@ -48,6 +51,32 @@ class Editor(Ia2Web):
 class EditorChunk(Ia2Web):
 	beTransparentToMouse = True
 
+class Math(Ia2Web):
+
+	def _get_mathMl(self):
+		from comtypes.gen.ISimpleDOM import ISimpleDOMNode
+		try:
+			node = self.IAccessibleObject.QueryInterface(ISimpleDOMNode)
+			# Try the data-mathml attribute.
+			attr = node.attributesForNames(1, (BSTR * 1)("data-mathml"), (c_short * 1)(0,))

```

Comment:
I think it makes it slightly better, in that I know what those bits are supposed to represent. I wasn't aware of the `type * 1` syntax.